ci/verify_bundle.py

```diff
@@ -310,6 +311,17 @@ def verify_torchscript(
         print("Provided TorchScript module is verified correctly.")
 
 
+def verify_configs(bundle_path: str, bundle: str):
+    test_config_file = f"test_{bundle}.py"
+    folder_name = "ci/config_tests"
+    if os.path.exists(os.path.join(folder_name, test_config_file)):
+        sys.path.append(folder_name)
+        module_name = f"test_{bundle}"
+        module = importlib.import_module(module_name)
+        module.test_bundle_configs(bundle_root=bundle_path)
+        print("all config files are verified correctly.")
+
+
 def verify(bundle, models_path="models", mode="full"):
     print(f"start verifying {bundle}:")
```

Comment:
`verify_bundle_keys` can be replaced with new workflow API.

Reply:
Hi @Nic-Ma , I re-think about it, and since it requires to use new bundle workflow, not all bundles can use the new API. Therefore, the current way will not be changed in this PR, and I will address it after preparing unit tests on all bundles first.